src/components/search-result-list/SearchResultList.tsx

```diff
@@ -10,21 +11,25 @@ export interface SearchResultListProps {
 const SearchResultList: React.FC<SearchResultListProps> = ({ resultItems }) => {
   const worksAreLoaded = Boolean(resultItems.length);
 
-  if (!worksAreLoaded) {
-    return null;
-  }
-
   return (
     <ul className="search-result-page__list my-32" data-cy="search-result-list">
-      {resultItems.map((item, i) => (
-        <li key={item.workId}>
-          <SearchResultListItem
-            item={item}
-            coverTint={getCoverTint(i)}
-            resultNumber={i + 1}
-          />
-        </li>
-      ))}
+      {/* Show skeleton search result items if no data is available yet */}
+      {!worksAreLoaded &&
+        [...Array(5)].map(() => (
```

Comment:
A comment about why 5 might be good. Is it a matter of having enough elements to fill out most user screens?